src/main/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/JobsService.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.jobscheduler.services.jobs;
+
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.jobscheduler.jobs.HttpCallJob;
+import uk.gov.hmcts.reform.jobscheduler.model.Job;
+
+import java.util.Date;
+import java.util.UUID;
+
+import static org.quartz.JobBuilder.newJob;
+import static org.quartz.TriggerBuilder.newTrigger;
+
+@Service
+public class JobsService {
```

Comment:
This class should be named `QuartzJobsService`.

Reply:
Disagree.
For same reason for example `DraftRepository` shouldn't be called `JdbcDraftRepository`